docker-compose.yml

```diff
@@ -1,16 +1,20 @@
-version: '2'
+version: '3'
 
 services:
+
   web:
-    build: .
+    container_name: bookit_web
+    image: builditdigital/bookit-web:latest
     ports:
-     - "8080:80"
+     - "80:80"
+    links:
+      - "server:bookitapi"
     environment:
-    - API_BASE_URL=http://localhost:8888
+      - BOOKIT_API_BASE_URL=/api
```

Comment:
Yeah, either I'm an idiot or Github is duplicating comment threads.  I'm going to stop now.